src/media/css/feed/feed.styl

```diff
@@ -71,11 +71,15 @@
     }
     .quote {
         color: $white;
-        left: -10px;
+        left: -5px;
```

Comment:
My mind is exploding looking at these left offsets. I don't really get why they're necessary, but they clearly are. Anyway, to stay on theme this could be `right: 5px` instead.